collect_app/src/test/java/org/odk/collect/android/views/TrackingTouchSliderTest.java

```diff
@@ -0,0 +1,45 @@
+package org.odk.collect.android.views;
+
+import android.widget.LinearLayout;
+
+import androidx.test.core.app.ApplicationProvider;
+import androidx.test.core.view.MotionEventBuilder;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.R;
+import org.robolectric.RobolectricTestRunner;
+
+import static android.view.MotionEvent.ACTION_DOWN;
+import static android.view.MotionEvent.ACTION_UP;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+@RunWith(RobolectricTestRunner.class)
+public class TrackingTouchSliderTest {
+
+    private TrackingTouchSlider slider;
+
+    @Before
+    public void setUp() {
+        ApplicationProvider.getApplicationContext().setTheme(R.style.Theme_Collect_Light);
+
```

Comment:
Oh this is very cool! It never occurred to me to just set the application context theme. This could probably help us speed up tests where we're building an `Activity` just to set the context theme. Maybe you could add this to `RobolectricHelpers` as `createThemedContext(int theme)` so we  can use it other places?

Reply:
Yes sure @seadowg